rfd/0223-install-docs.md

```diff
@@ -0,0 +1,208 @@
+3. Clicks each sidebar item and finds "Installation" under "Introduction"
+4. Clicks "Installation," then "Linux"
+5. Sees instructions for installing a Teleport cluster, continues to scroll down to agent
+6. Runs an install script that results in an auto-updating agent that is not configured
+7. Sees that they need to manually enroll the agent, clicks "Enroll Resources"
+8. Clicks "Linux Servers (section)"
+9. Sees a page with four different guides that are named very similarly
+10. Clicks the "Getting Started Guide", or may get directed there by clicking one of the other guides
+11. Sees the install script instructions again, along with other outdated installation instructions
+12. Finds and executes manual joining instructions, which are unnecessary if the Web UI is used.
+
+The Teleport Downloads page has a similar issue that is actively being addressed.
+
+This RFD proposes that we add one new top-level "Platform" section, with three subsections:
+
+1. "Installation"
+2. "Upgrading"
+3. "Operations"
+
+These sections would cover all Teleport components.
+Installation sections will cover setup for Managed Updates, and link to more details Upgrading docs where appropriate.
+
+Notably, the Platform section differs from the marketing-branded categories for our product features because it is cross-cutting,
+operational, and (in most cases) does not involve in-product UX workflows.
+Users install Teleport first, ensure they have a way to upgrade it, ensure they have day-two operations like backups
+configured, and then they (or different end-users) dive into the product itself.
+
+A complete user story for a Teleport installation workflow may need to involve in-product configuration.
+For example, it is common to configure roles after installing a self-hosted cluster.
+As we cannot repeat these instructions for every combination of component and target platform, we can rely on cross-linking to ensure
+that the user is guided to the correct page regardless of their starting point.
+
+## Details
+
+### Overview of All Components with Customer-Managed Operational Lifecycle
+
+Teleport includes a large number of independently installable components that require dedicated installation
+instructions for an even larger number of target platforms:
+
+1. Cluster Installation
+   1. Self-Hosted on Linux
+   2. Self-Hosted on Kubernetes
+      1. Digital Ocean
+      2. Helm
+      3. Helm with ArgoCD
+      4. EKS
+      5. GKE
+      6. AKS
+      7. IBM Cloud
+      8. Helm “Custom”
+   3. Self-Hosted on Docker
+   4. Self-Hosted on EC2
+   5. Self-Hosted on GCP
+   6. Self-Hosted on OCI
+   7. Self-Hosted on Azure
+   8. Self-Hosted via Source Code
+2. Agent Installation
+   1. Agent on Kubernetes
+   2. Agent on Docker
+   3. Agent on EC2
+      1. Teleport AMI
+      2. Custom AMI
+      3. Via Deploy Service / AWS OIDC
+   4. Agent on GCP
+   5. Agent on OCI
+   6. Agent on Azure
+3. Client Installation
```

Comment:
You don't show moving these as part of the action plan. Are you proposing that these would move from the user guides section to the platform/installation section?